index.js

```diff
@@ -378,4 +378,17 @@ export const mentorshipTeam = [
 			},
 		],
 	},
+	{
+    name: 'Khang Huu Nguyen',
```

Comment:
hmmm not sure why but it looks like the code is not indented as properly as the above

Reply:
I think it's because I'm using space and the original code is using tabs